test/unit/logcat-specs.js

```diff
@@ -12,7 +12,8 @@ describe('logcat', async () => {
   let adb = {path: 'dummyPath', defaultArgs: []};
   let logcat = new Logcat({adb, debug: false, debugTrace: false});
   describe('startCapture', withMocks({teen_process}, (mocks) => {
-    it('should correctly call subprocess and should resolve promise', async () => {
+    // Because of  ExpectationError: Unexpected call: resolve(...) on Travis
```

Comment:
Shouldn't it be a separate PR?

Reply:
ah, yes. I'll create another PR to skip them and rebase after this PR after merge it.